docs/PyPiPublishing.md

```diff
@@ -0,0 +1,159 @@
+[![GitHub Release](https://img.shields.io/github/v/release/openvinotoolkit/nncf?color=green)](https://github.com/openvinotoolkit/nncf/releases)
+[![Website](https://img.shields.io/website?up_color=blue&up_message=docs&url=https%3A%2F%2Fdocs.openvino.ai%2Flatest%2Fopenvino_docs_model_optimization_guide.html)](https://docs.openvino.ai/nncf)
+[![Apache License Version 2.0](https://img.shields.io/badge/license-Apache_2.0-green.svg)](LICENSE)
+[![PyPI Downloads](https://static.pepy.tech/badge/nncf)](https://pypi.org/project/nncf/)
+
+# Neural Network Compression Framework (NNCF)
+
+Neural Network Compression Framework (NNCF) provides a suite of post-training
+and training-time algorithms for optimizing inference of neural networks in
+[OpenVINO&trade;](https://docs.openvino.ai) with a minimal accuracy drop.
+
+
+NNCF is designed to work with models from [PyTorch](https://pytorch.org/),
+[TensorFlow](https://www.tensorflow.org/), [ONNX](https://onnx.ai/) and
+[OpenVINO&trade;](https://docs.openvino.ai).
+
+The framework is organized as a Python package that can be built and used
+in a standalone mode. Its architecture is unified to make it easy to
+add different compression algorithms for both PyTorch and TensorFlow deep
+learning frameworks.
+
+
+NNCF provides [samples](#demos-tutorials-and-samples) that demonstrate the
+usage of compression algorithms for different use cases and models. See
+compression results achievable with the NNCF-powered samples on the
+[NNCF Model Zoo page](https://github.com/openvinotoolkit/nncf/blob/develop/docs/ModelZoo.md).
+
+For more information about NNCF, see:
+- [NNCF repository](https://github.com/openvinotoolkit/nncf)
+- [user documentation](https://docs.openvino.ai/nncf)
+- [NNCF API documentation](https://openvinotoolkit.github.io/nncf/autoapi/nncf/)
+- [usage examples](https://github.com/openvinotoolkit/nncf/tree/develop/docs/usage)
+- [Notebook tutorials](https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/notebooks/README.md#model-training)
+
+
+
+
+## Table of contents:
+
+- [Key Features](#key-features)
+- [Installation](#installation-guide)
+- [Third-party integration](#third-party-repository-integration)
+- [NNCF Model Zoo](#nncf-compressed-model-zoo)
```

Comment:
Those links would not work on PyPI because the `<a>` tag is absent in the corresponding paragraph.
Ref: https://github.com/pypa/readme_renderer/issues/169#issuecomment-808577486

Reply:
hopefully, they will now - added the tag